salesagility/SuiteCRM
# Make Business Hours less mysterious

## Description
There is no Issue for this PR, I'm assuming that isn't strictly necessary.

## Motivation and Context
Nobody knows what "Business Hours" does. Yet after discussing this in the forums with a user that actually needed this feature, I tried a few tests to understand the feature, and here I basically improve the messages to the user to be more helpful.

## How To Test This
Check `Admin` screen, new message there.
Click `Business Hours`, new message there also.

I'm pushing this `hotfix` because it's such a simple fix I don't think it needs to wait until a development release, we can look at this as a bugfix in a message :-)

P.S. - I also took the chance to remove a few full-stops on the Admin screen descriptions, for consistency with the rest of the messages
